showcase/.ember-cli

```diff
@@ -8,8 +8,8 @@
   "disableAnalytics": true,
 
   /**
-  Setting `isTypeScriptProject` to true will force the blueprint generators to generate TypeScript
-  rather than JavaScript by default, when a TypeScript version of a given blueprint is available.
+    Setting `isTypeScriptProject` to true will force the blueprint generators to generate TypeScript
+    rather than JavaScript by default, when a TypeScript version of a given blueprint is available.
   */
-  "isTypeScriptProject": false
+  "isTypeScriptProject": true
```

Comment:
@alex-ju since we are enabling the blueprints for the showcase as TS, do you think we should also update the `showcase/blueprints/hds-component/files/src/components/hds/__name__/index.js` file to remove the `Index` from the class name (in this PR, I mean)?

Reply:
yes, [we have an action for this already](https://docs.google.com/document/d/1twnk39hZ7p8oZYrld_FYsdiuUb6yn_weA46KnhF4nu4/edit?disco=AAABNdTuLbc)